package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "use-http",
-  "version": "0.1.22",
-  "main": "src/main.js",
+  "version": "0.1.23",
+  "main": "build/main.js",
+  "peerDependencies": {
```

Comment:
Now that we're shipping the transpiled bundle, react should really be a peer dependency for this sort of library.